<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addModelFrm</class>
 <widget class="QWidget" name="addModelFrm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加模型</string>
  </property>
  <widget class="QPushButton" name="cancleBtn">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modelName">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>17</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>模型名称：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>281</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>加载</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>54</y>
     <width>276</width>
     <height>211</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>模型路径</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>位置</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>经度：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>47</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>纬度：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>75</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>高度：</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="setModelLong">
      <property name="geometry">
       <rect>
        <x>54</x>
        <y>15</y>
        <width>175</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>116.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="setModelLati">
      <property name="geometry">
       <rect>
        <x>54</x>
        <y>44</y>
        <width>175</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>8</number>
      </property>
      <property name="minimum">
       <double>-90.000000000000000</double>
      </property>
      <property name="maximum">
       <double>90.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>39.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="setModelAlti">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>72</y>
        <width>174</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>8</number>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>32</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>选择路径：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectModelPathBtn">
     <property name="geometry">
      <rect>
       <x>231</x>
       <y>28</y>
       <width>24</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="modelPath">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>155</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>缩放系数</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>35</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>X轴：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>62</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Y轴：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Z轴：</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="setModelScaleX">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>120</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="setModelScaleY">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>59</y>
        <width>120</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="setModelScaleZ">
      <property name="geometry">
       <rect>
        <x>71</x>
        <y>87</y>
        <width>120</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>模型朝向</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>19</y>
       <width>248</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>38</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>X轴：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>67</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Y轴：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>93</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Z轴：</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="setModelRouteX">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>33</y>
        <width>172</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="setModelRouteY">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>62</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="setModelRouteZ">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
